catalog/dags/providers/provider_api_scripts/europeana.py

```diff
@@ -157,6 +158,17 @@ def _get_filetype(self, item_data: dict) -> str:
     def _get_filesize(self, item_data: dict) -> int:
         return item_data.get("ebucoreFileByteSize")
 
+    @staticmethod
+    def _get_creator(self, data: dict) -> str | None:
+        creators = data.get("dcCreator", [])
+        if isinstance(creators, list):
+            if not creators:  # Explicitly handle empty list
+                return None
+            return ",".join(creators)
```

Comment:
```suggestion
            return ", ".join(creators)
```
The first version, with the space, was better.